plugins/dell/fu-self-test.c

```diff
@@ -485,6 +485,7 @@ main (int argc, char **argv)
 	sysfsdir = fu_common_get_path (FU_PATH_KIND_SYSFSDIR_FW);
 	g_setenv ("FWUPD_UEFI_ESP_PATH", sysfsdir, TRUE);
 	g_setenv ("FWUPD_DELL_FAKE_SMBIOS", "1", FALSE);
+	g_setenv ("FWUPD_IGNORE_SYSTEMD", "1", FALSE);
```

Comment:
Not actually used

Reply:
Agree, that's what I'm using in another branch. I've removed it here.